### ISSUE TITLE
Model Structure Visualization & Transfer Learning

### ISSUE THREAD

**Issue by CHANG-Shaole** [deepchem/deepchem#3082]
## ❓ Questions & Help

Hi, thanks for such a great python package for computational chemistry.
There I want to ask some questions about how to better use it.
I used a graph model created by `deepchem.models.GraphConvModel` class, and I set the parameters `n_tasks=1, dropout=0.5, optimizer=dc.models.optimizers.Adam(learning_rate=0.001), batch_size=128, graph_conv_layers=[32, 32], dense_layer_size=64, number_atom_features=75, mode="regression"` to train the model. The first question is how I can visualize the model to see what actually the layers' structure is, just like the `torch.model.summary`.
After training the first model, I also want to use another dataset to train the model using transfer learning technology. So the second question is how I can change the model's structure, such as how can I add another dense layer to the model at the end. The last question is how to freeze the weights of certain layers, you know when using the transfer learning technology, sometimes we need to freeze some model weights.
Anyone who answers the questions will be appreciated.
Thanks in advance!
